### PR TITLE
Book Detail I and II - Texts and Image

### PR DESCRIPTION
**Pull request by joacomiguens**
### Summary

- Book detail card texts added.

- Book detail card cover added.

- Dark-grey color for genre and  content set.

- Roboto font set globally.

### Screenshots

image

### Trello Cards

You can find it [here](https://trello.com/c/uM6hBCxD/26-detalle-de-libro-i-textos) and [here](https://trello.com/c/vjutCwNo/27-detalle-de-libro-ii-imagen)

### CODE REVIEW

File: scss/_detail.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+%content {
+  color: $dark-grey;
+  font-size: 16px;
+}
+
+.detail {
+  display: flex;
+  width: 100%;
+  max-width: 600px;
+  padding: 16px;
```
Comment (ebedoyaalzate) [inline scss/_detail.scss:15]: In wolox we sort properties by alphabetical order.

Comment (ebedoyaalzate) [inline scss/_detail.scss:14]: In wolox we sort properties by alphabetical order.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (ebedoyaalzate) [inline .gitignore]: Add EOF

File: scss/_detail.scss (hunk 1 above)
Comment (joacomiguens) [reply scss/_detail.scss:14]: Done 😃

File: .gitignore (hunk 2 above)
Comment (joacomiguens) [reply .gitignore]: Done 😃

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="detail">
+      <div class="detail__cover">
```
Comment (mcallegari10) [inline index.html]: I think this `div` is not needed at all

File: scss/_detail.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  width: 100%;
+
+  &__cover {
+    display: flex;
+    margin-right: 32px;
+
+    &__img {
+      width: 250px;
```
Comment (mcallegari10) [inline scss/_detail.scss]: You can set the width like:

```css
max-width: 250px;
width: 100%;
```

That way, the image will occupy the maximum space it has 😄

File: scss/_detail.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  }
+
+  &__info {
+    display: inline-flex;
```
Comment (mcallegari10) [inline scss/_detail.scss]: I think this is just `flex` 😄

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (joacomiguens) [reply index.html]: Done 😄

File: scss/_detail.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_detail.scss]: Done 😄

File: scss/_detail.scss (hunk 5 above)
Comment (joacomiguens) [reply scss/_detail.scss]: Done 😄

**Review by ebedoyaalzate — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**